packages/adapters/ethers/src/tests/client.test.ts

```diff
@@ -26,13 +37,16 @@ vi.mock('@reown/appkit-wallet', () => ({
   }
 }))
 
-vi.mock('@reown/appkit-utils', () => {
+vi.mock('@reown/appkit-utils', async importOriginal => {
+  const actual = await importOriginal()
```

Comment:
To not mock `CaipNetworkUtil` to extend networks as expected on test environment